ui/pages/confirmations/components/confirm/nav/nav.tsx

```diff
@@ -134,6 +136,7 @@ const Nav = () => {
       <Button
         borderRadius={BorderRadius.XL}
         className="confirm_nav__reject_all"
+        data-testid="confirm-nav__reject-all"
```

Comment:
Just wondering if its possible to avoid adding these `data-testid`.

Reply:
data-testid helps with improved test readability, flakiness and makes it easier to locate elements hence always nice to have.